.travis.yml

```diff
@@ -10,41 +10,15 @@ env:
     - secure: kCOLgEveEDprSL3fOBeCc1/nmHVYEcNh/lRRWiJywFehOzKBTBF40pAWX7JtsWUPG8aPKGYFEE4sjYd+DF2DRknbuB2ZfAEgs9Xp9h5Mx1KztJYNiCfEnLciBBRDXA7t1M3TLKyT3ptA+Kk307cH2QHv9sUUnhnf0IofJyQhQvA=
   
 jdk:
-  #- openjdk7
-  - oraclejdk8
+  - openjdk8
+  - openjdk11
 
-before_install:
-  # required for testing jdk 11
-  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh -O $TRAVIS_BUILD_DIR/install-jdk.sh
-  - chmod u+x $TRAVIS_BUILD_DIR/install-jdk.sh
-
-install:
-  # compile and test java 8
-  - mvn -B -V clean test
-
-script:
-  # compile tests with java 8
-  - jdk_switcher use oraclejdk8
-  - mvn -B clean install
-  # test java 9 (only available via oraclejdk9)
-  - jdk_switcher use oraclejdk9
-  - mvn -v && mvn -B test
-  # test java 11
-  # see also https://docs.travis-ci.com/user/languages/java/#using-java-10-and-later
-  # and https://github.com/sormuras/bach#install-jdksh (because default travis docs didn't work)
-  - export JAVA_HOME=$HOME/openjdk11
-  - $TRAVIS_BUILD_DIR/install-jdk.sh --feature 11 --target $JAVA_HOME
-  - mvn -v && mvn -B test
-  # install and publish snapshot with java 8
-  # not via `after_success` but directly in `script`, since a failure would be silently ignored in `after_success`, which
-  # is documented in https://docs.travis-ci.com/user/customizing-the-build#Breaking-the-Build
-  - jdk_switcher use oraclejdk8
-  - if [ $TRAVIS_BRANCH = "master" ] && [ $TRAVIS_PULL_REQUEST = "false" ] && [ $IS_SNAPSHOT ]; then
```

Comment:
Removing all this is not what we want I'd say.

The build in the past built/compiled with java 8, and then ran tests with java9 and java11, to verify if the published artifact (built with java8) works with java9 and java11.

AFAICS, this is no longer the case with this change.

My approach for this oracle java 8 issue would have been to add
```
dist: trusty
```
following  https://travis-ci.community/t/solved-oraclejdk8-installation-failing-still-again/3428 and https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment

Not sure though if this works here

Reply:
@magro oraclejdk8 is EOL, swapped to openjdk8 (still receiving updates - LTS), 9 and 10 are eol, no updates, included openjdk11 for tests, by default travis installs the artifacts and runs all unit tests - https://docs.travis-ci.com/user/languages/java#projects-using-maven , then there's an after script hook to deploy a snapshot (built with 8) to sonatype like the previous setup.

Reply:
https://travis-ci.org/EsotericSoftware/kryo/builds/612316865?utm_source=github_status&utm_medium=notification shows the results of these travis configuration changes.

Reply:
@magro so essentially you'd like me to rollback the jdk_switcher changes to test a java8 binary on different java runtimes?

Reply:
@magro ptal I rolled back the travis changes.